recipe/meta.yaml

```diff
@@ -11,7 +11,7 @@
 {% set blas_minor = build_num + (version_minor|int - 8) * 100 %}
 
 # versions of pinned blas implementation flavours
-{% set blis_version = "0.9.0" %}
+{% set blis_version = "2.0" %}
```

Comment:
I dislike changing lengths of version numbers; but it appears that blis did so after moving away from 0ver